test/gas/gas.js

```diff
@@ -0,0 +1,563 @@
+const assert = require("assert");
```

Comment:
Why are some tests split into their own files, while the bulk of tests remain in this one large file?

Reply:
The test counts were preserved for each of the test files.